app/helpers/application_helper.rb

```diff
@@ -130,4 +128,12 @@ def pontiiif_server
     Rails.application.config.respond_to?(:pontiiif_server) && Rails.application.config.pontiiif_server
   end
 
+  def language_attrs(collection)
+    collection = collection
```

Comment:
Redundant?

Reply:
Whoops - totally redundant.  I was using an instance variable initially, and then forgot when I switched it.